src/test_typing_extensions.py

```diff
@@ -5718,8 +5717,7 @@ def test_typing_extensions_defers_when_possible(self):
                     getattr(typing, item))
 
     def test_typing_extensions_compiles_with_opt(self):
-        file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                 'typing_extensions.py')
+        file_path = typing_extensions.__file__
```

Comment:
dumb question, why is this needed?

Reply:
Probably because in the distribution tests above, `typing_extensions.py` gets deleted (to test the installed version).